### PR TITLE
Updating the help messages for AzCopyV10

### PR DESCRIPTION
**Pull request by artemuwka**

### CODE REVIEW

File: cmd/helpMessages.go [hunk 1]
```diff
@@ -28,62 +28,62 @@ Copies source data to a destination location. The supported pairs are:
...
+const copyCmdExample = `Upload a single file using OAuth authentication. Please use 'azcopy login' command first if you aren't logged in yet:
+- azcopy cp "/path/to/file.txt" "https://[account].blob.core.windows.net/[container]/[path/to/blob]"
 
-Upload a single file with OAuth token, please use login command first if not yet logged in:
-  - azcopy cp "/path/to/file.txt" "https://[account].blob.core.windows.net/[container]/[path/to/blob]"
+Upload a single file with a SAS:
+  - azcopy cp "/path/to/file.txt" "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]"
 
-Upload a single file through piping(block blob only) with SAS:
+Upload a single file with a SAS using a pipeline (block blobs only):
```
Comment (zezha-msft) [inline cmd/helpMessages.go]: Not pipeline, it's piping

Comment (zezha-msft) [inline cmd/helpMessages.go]: it should be piping not pipeline

File: cmd/helpMessages.go [hunk 2]
```diff
@@ -167,21 +167,21 @@ const makeCmdExample = `
...
+const removeCmdLongDescription = `Delete blobs or files from Azure Storage.`
 
 // ===================================== SYNC COMMAND ===================================== //
-const syncCmdShortDescription = "Replicates source to the destination location"
+const syncCmdShortDescription = "Replicate source to the destination location"
 
 const syncCmdLongDescription = `
-Replicates source to the destination location. The last modified times are used for comparison. The supported pairs are:
-  - local <-> Azure Blob (SAS or OAuth authentication)
+Replicate a source to a destination location. The last modified times are used for comparison, the file is skipped if the last modified time in the destination is more recent. The supported pairs are:
```
Comment (zezha-msft) [inline cmd/helpMessages.go]: I've updated this in my PR too. But it looks good.

**Comment by zezha-msft** [comment]
@artemrd thanks! Looks good to me. Please merge whenever you're ready.

**Comment by artemuwka** [comment]
#sign-off

**Pull request merged**